__tests__/integration/wallet-get.spec.js

```diff
@@ -0,0 +1,138 @@
+require('dotenv').config();
+const request = require('supertest');
+const {expect} = require('chai');
+const sinon = require('sinon');
+const chai = require('chai');
+const server = require('../../server/app');
+chai.use(require('chai-uuid'));
+const {registerAndLogin, clear, feedSubWallets} = require('../utils/testUtils');
+const walletAInfo = require('../mock-data/wallet/walletA.json');
+const walletBInfo = require('../mock-data/wallet/walletB.json');
+const wallets = require('../mock-data/wallet/wallets.json');
+
+
+describe('Wallet: Get wallets of an account', () => {
+    let walletA;
+    let walletB;
+
+    before(async () => {
+        await clear();
+    })
+
+    beforeEach(async () => {
+        walletA = await registerAndLogin(walletAInfo);
+        // what registerAndLogin returned?
+        // walletA = {
+        //                  name: <string>,
+        //                  password: <string>,
+        //                  salt: <string>,
+        //                  logo_url: null,
+        //                  created_at: <time>,
+        //                  apiKey: <string>,
+        //                  token: <string> (the bearer token)
+        //                 }
+        walletB = await registerAndLogin(walletBInfo);
+        await feedSubWallets(walletA.id, wallets);
+    });
+
+    afterEach(async () => {
+        sinon.restore();
+        await clear();
```

Comment:
There is another await clear(). I think you can just put it as the first statement in the beforeEach function, that way there won't be any duplicates

Reply:
I do this because I can ensure the database is empty before the whole test starts. And also, I can clean the database after all tests are finished. so that the developer's database does not be contaminated by test data. if we just use beforeEach to clean the database, then after the last test is finished, the data will remain in the database(mysophobia???? lol). do you think this matters? If not, I will change it.